physionet-django/console/views.py

```diff
@@ -2952,6 +2952,21 @@ def event(request):
                    })
 
 
+@permission_required('user.view_all_events', raise_exception=True)
+def archive_event(request):
+    """
+    List of archived events
+    """
+    now = timezone.now()
```

Comment:
Not sure that it is necessary to assign the `now` variable here. `Event.objects.filter(end_date__lte=timezone.now())` is just as readable.